cypress/page-objects/student-path.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+
+import { contains } from "cypress/types/jquery";
+
+export class StudentPath {
+  goToSubjectPage(): void {
+    cy.get(".font-medium > .inline-flex")
+    cy.contains("Try it out").click().wait(1000);
+      }
+  checkSubjectPage(): void {
+    cy.get("h1.text-4xl").should("have.text", "Subjects page").wait(1000);
+  }
+   selectSubject(subjectName: string): void {
+    cy.get(".p-6.my-4")
+     .find(".text-4xl.my-1.font-semibold.capitalize")
+     .contains(subjectName)
+     .get(':nth-child(1) > .inline-block > .flex > .pr-4').click()
+   }
+   checkUrl(env:): void {
+     cy.url().should("include", "/subjects/Remote%20work").wait(1000);
+       }
```

Comment:
I would include this assertion in the method 'goToSubjectPage' right after:
`get(':nth-child(1) > .inline-block > .flex > .pr-4').click()`. No need for separate method for assertion.

Reply:
Also, what is the purpose to put the argument `env:` into this method? And in TS this argument should be `env: string`.